lib/SILGen/SILGenDistributed.cpp

```diff
@@ -133,23 +133,61 @@ static void emitDistributedIfRemoteBranch(SILGenFunction &SGF, SILLocation Loc,
 // ==== ------------------------------------------------------------------------
 // MARK: local instance initialization
 
+static SILArgument *findFirstDistributedActorSystemArg(SILFunction &F) {
+  auto *module = F.getModule().getSwiftModule();
+  auto &C = F.getASTContext();
+
+  auto *transportProto = C.getProtocol(KnownProtocolKind::DistributedActorSystem);
```

Comment:
nitpick "system" transport was ancient wording.

```suggestion
  auto *DAS = C.getDistributedActorSystemDecl();
```

or `systemProto` if you prefer 👍  recently started using `DAS` `DA` for the protocol types.